myhomework2.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+// не звертайте на цю функцію уваги
+// вона потрібна для того щоб правильно зчитати вхідні дані
+function readHttpLikeInput()
+{
+    $f = fopen('php://stdin', 'r');
+    $store = "";
+    $toread = 0;
+
+    while ($line = fgets($f)) {
+        $store .= preg_replace("/\r/", "", $line);
+
+        if (preg_match('/Content-Length: (\d+)/', $line, $m))
+            $toread = $m[1] * 1;
+
+        if ($line === PHP_EOL)
+
+            break;
+    }
+
+    if ($toread > 0)
+        $store .= fread($f, $toread);
+
+    return $store;
+}
+
+$contents = readHttpLikeInput();
+
+function parseTcpStringAsHttpRequest($string)
+{
+
+    return ["method" => getMethod($string),
```

Comment:
looks strange. it should look like this in such cases:
[
    value,
    value2,
    ...
]

Reply:
it trimmed spaces, I used for formatting. you should have tabulation there.